### PR TITLE
Add checks to 3d bubble problem. Add ClimaCoreVTK and ClimaCorePlots to Project

### PR DESCRIPTION
**Pull request by akshaysridhar**
add checks in 3d bubble problem
	modified:   ../Project.toml
	modified:   ../src/Models/Nonhydrostatic3DModels/equations_thermodynamics.jl
	modified:   test_cases/initial_conditions/dry_rising_bubble_3d.jl
	modified:   test_cases/run_dry_rising_bubble_3d.jl

### CODE REVIEW

**Comment by akshaysridhar** [comment]
bors try

**Comment by akshaysridhar** [comment]
bors try

File: test/test_cases/run_dry_rising_bubble_3d.jl [hunk 1]
```diff
@@ -31,31 +35,169 @@ function run_dry_rising_bubble_3d(
...
+            set!(simulation, :thermodynamics, ρθ = ρθ)
+            u = simulation.integrator.u
+            ∫ρ_0 = sum(u.base.ρ)
+            ∫ρθ_0 = sum(u.thermodynamics.ρθ)
+            step!(simulation)
+            u = simulation.integrator.u
+            # perform regression check
+            current_min = 299.9999999523305
+            current_max = 300.468563373248
+            θ = u.thermodynamics.ρθ ./ u.base.ρ
```
Comment (bischtob) [inline test/test_cases/run_dry_rising_bubble_3d.jl:94]: Can we have a slightly more informative regression test? As I see it the bubble is evolved for one time step and therefore doesn't change very much. So we are effectively regression testing the initial condition here.

File: test/test_cases/run_dry_rising_bubble_3d.jl [hunk 2]
```diff
@@ -31,31 +35,169 @@ function run_dry_rising_bubble_3d(
...
-        @test_throws ArgumentError set!(simulation, quack = ρ)
-        @test_throws ArgumentError set!(simulation, ρ = "quack")
+            step!(simulation)
 
-        # test successful integration
-        @test step!(simulation) isa Nothing # either error or integration runs
-    elseif mode == :regression
-        # TODO!: Implement meaningful(!) regression test
+            current_min = 237082.14581933746
+            current_max = 252441.54599695574
```
Comment (bischtob) [inline test/test_cases/run_dry_rising_bubble_3d.jl:121]: This is better

File: test/test_cases/run_dry_rising_bubble_3d.jl [hunk 3]
```diff
@@ -31,31 +35,169 @@ function run_dry_rising_bubble_3d(
...
+            ENV["GKSwstype"] = "nul"
+            Plots.GRBackend()
+            # make output directory
+            path = joinpath(@__DIR__, "output_validation")
+            mkpath(path)
+            ClimaCoreVTK.writevtk(joinpath(path, "test"), θ)
+            @test true # check is visual
+        end
+        # Total Energy Prognostic
+        @testset "Validation: Total Energy Model" begin
```
Comment (bischtob) [inline test/test_cases/run_dry_rising_bubble_3d.jl:170]: Could you add a few more verbose comments explaining what we are testing?

**Review by bischtob — CHANGES_REQUESTED** [review]
This is great. If you could address the minor changes, that would be great.

**Pull request merged**